src/Builder/ComposerClassLoader/ComposerClassLoader.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\ComposerClassLoader;
+
+use Composer\Autoload\ClassLoader;
+
+final class ComposerClassLoader
```

Comment:
I'm not sure this class should exist. 
1. It is ClassLoader but exists apart from ClassLoader directory. Doesn't make any sense.
2. If it is a class loader, then it should be a subtype of the ClassLoaderInterface. Otherwise, why do we have that hierarchy, right?
3. It is used only in places where the ClassLoader class is used, so why just not expand ClassLoader instead?

Reply:
It's easy to use it like static class. I did it on purpose for ease of use. Has method we can use everywhere

Reply:
We need to do refactoring. It's hard to use dependency now. Needs to move all functionality that uses static code to static helper. I guess I created one refactoring ticket to move some functionality to abstract layer

Reply:
refactored